### PR TITLE
upload package to artifact and OSS nightly

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          filepath=$(find build/cpack_output -type f \( -iname \*.deb -o -iname \*.rpm \))
+          ./ci/scripts/upload-oss.sh \
+            OSS_ENDPOINT=${{ secrets.OSS_ENDPOINT }} \
+            OSS_ID=${{ secrets.OSS_ID }} \
+            OSS_SECRET=${{ secrets.OSS_SECRET }} \
+            filepath=$filepath \
+            nightly=true
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.os }}-nightly
```
Comment (yixinglu) [inline .github/workflows/package.yml:44]: v2-nightly

**Review by yixinglu — COMMENTED** [review]
Try to search whether there's some action like oss upload.

File: .github/workflows/package.yml (hunk 1 above)
Comment (jievince) [reply .github/workflows/package.yml]: fixed

**Comment by jievince** [comment]
> Try to search whether there's some action like oss upload.

OK

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -26,19 +26,34 @@ jobs:
     container:
       image: vesoft/nebula-dev:${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```
Comment (laura-ding) [inline .github/workflows/release.yml:29]: v2 is not supported under centos6

Comment (jievince) [reply .github/workflows/release.yml]: Ok, fixed

File: .github/workflows/package.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+name: package
+
+on:
+  push:
+    branches:
+      - 'v[0-9]+.*'
```
Comment (yixinglu) [inline .github/workflows/package.yml:6]: nightly package should based on master branch.

Comment (jievince) [reply .github/workflows/package.yml:6]: OK

**Review by nevermore3 — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**